### PR TITLE
docs: Add Tensorflow documentation

### PR DESCRIPTION
**Pull request by GabyCT**
This PR adds the Tensorflow documentation as we already have Tensorflow
benchmark as part of the Kata Containers metrics.

### CODE REVIEW

File: metrics/machine_learning/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Kata Containers Tensorflow Metrics
+
+Kata Containers provides a series of performance tests using the
+TensorFlow reference benchmarks (tf_cnn_benchmarks).
+The tf_cnn_benchmarks containers TensorFlow implementations of several
```
Comment (jodh-intel) [inline metrics/machine_learning/README.md:5]: Nit: I think you might need to put the `tf_* terms in back ticks to keep the spell checker happy.

**Review by jodh-intel — APPROVED** [review]
Thanks @GabyCT.

lgtm but see the comment about the spell-checker if that fails.

**Review by cmaf — APPROVED** [review]

**Pull request merged**